packages/eslint-plugin-office-addins/src/rules/office-ready.ts

```diff
@@ -0,0 +1,30 @@
+import { TSESTree } from "@typescript-eslint/typescript-estree";
+
+export = {
+  name: "office-ready",
+  meta: {
+    type: <"problem" | "suggestion" | "layout">"suggestion",
+    messages: {
+      officeOnReady: "Prefer calling Office.onReady() instead of Office.initialize",
+    },
+    docs: {
+      description: "It is a bad practice to call Office.initialize instead of Office.onReady",
+      category: <"Best Practices" | "Stylistic Issues" | "Variables" | "Possible Errors">"Best Practices",
```

Comment:
I would describe it more like "Office.onReady() is more flexible the Office.initalize" rather then indicating it's a "bad practice".

This will likely be updated later when we get better messages.